lib/checks/aria/has-global-aria-attribute.json

```diff
@@ -0,0 +1,11 @@
+{
+	"id": "has-global-aria-attribute",
+	"evaluate": "has-global-aria-attribute-evaluate",
+	"metadata": {
+		"impact": "minor",
+		"messages": {
+			"pass": "Element has global ARIA attribute",
```

Comment:
It would be helpful to the user to know which attribute(s) are the global ones. In the evaluate function, if you call `this.data()` and pass it an array of the global aria attributes that the element has, you can then use `${data.values}` and the [singular/plural structure in the message](https://github.com/dequelabs/axe-core/blob/develop/doc/check-message-template.md#singular-and-plural-messages) to have them listed out for the user:

```js
// has-global-aria-attribute-evaluate.js 
const globalAttrs = getGlobalAriaAttrs().filter(attr => virtualNode.hasAttr(attr));
this.data(globalAttrs);
return globalAttrs.length > 0;

// has-global-aria-attribute.json
{
  "messages": {
    "pass": {
      "singular": "Element has global ARIA attribute: ${data.vaules}",
      "plural": "Element has global ARIA attributes: ${data.vaules}"
    }
  }
}
```

Reply:
Fixed! Thanks for this :)